lxd/seccomp/seccomp.go

```diff
@@ -548,6 +727,11 @@ func seccompGetPolicyContent(s *state.State, c Instance) (string, error) {
 			// multiple syscalls.
 			policy += seccompBlockNewMountAPI
 		}
+
+		if shared.IsTrue(config["security.syscalls.intercept.bpf"]) &&
```

Comment:
Shouldn't that do the same as mount and intercept the syscall as soon as `intercept.bpf` is true, then let you change what type is allowed without needing to restart the container to update seccomp?

Reply:
Good idea! Done.